docs/topics/mission-rest-api/assembly_building-and-deploying-the-booster-to-openshiftlocal.adoc

```diff
@@ -0,0 +1,23 @@
+
+[#building-and-deploying-the-rest-api-booster-to-openshiftlocal]
+= Building and Deploying the {name-mission-http-api} Booster to {OpenShiftLocal}
+
+You have two options how to execute the {name-mission-http-api} booster locally on {OpenShiftLocal}:
```

Comment:
Consider rephrasing to:

You have two options for to executing the {name-mission-http-api} booster locally on {OpenShiftLocal}